jlibra-examples/src/main/java/dev/jlibra/example/MintExample.java

```diff
@@ -29,17 +27,16 @@ public static void main(String[] args) throws IOException {
         String toAddress = "045d3e63dba85f759d66f9bed4a0e4c262d17f9713f25e846fdae63891837a98";
         long amountInMicroLibras = 10L * 1_000_000L;
 
-        URL faucet = new URL(
-                String.format("http://faucet.testnet.libra.org?amount=%d&address=%s", amountInMicroLibras, toAddress));
+        HttpResponse<String> response = Unirest.post("http://faucet.testnet.libra.org")
```

Comment:
I quite liked that we don't use many third party libraries. Is there a point to add UniRest just to make one POST?

Reply:
I thought Sauli is right that as it is restructured now, the core does still have no other dependencies, the dependencies are just in jlibra-examples. Not in core, so users can just not use examples as a dependency - and they most likely won't do this. So to me that's fine as long as it's not in core. POST in plain Java is a lot of boilerplate.

Reply:
Yes, the motivation for avoiding dependencies for me was to limit the number of transitive deps for other projects. But as the examples and integration tests are not used in other projects I don't see a problem with this.